src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -4,16 +4,18 @@ import {
 	getExtractionProvider,
 } from '../../providers/extraction_providers/extractionProviderFactory';
 import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
-import { BusinessInferences, BusinessInfoInput } from '../generatedTypes';
+import { BusinessInferences, QueryGetBusinessInfoArgs } from '../generatedTypes';
 
-export async function getBusinessInfo(businessInfo: BusinessInfoInput): Promise<BusinessInferences> {
+export async function getBusinessInfo(_: any, args: QueryGetBusinessInfoArgs): Promise<BusinessInferences> {
```

Comment:
Apparently this is what the argument structure of a resolver looks like: 
![image](https://user-images.githubusercontent.com/23726221/115293852-d1c2d280-a125-11eb-8088-7842bba888fd.png)

More info can be found [here](https://www.apollographql.com/docs/tutorial/resolvers/).

Reply:
It seems like the `parent` field is undefined when I did some testing.

Reply:
The `parent` is defined if you're resolving a sub-field (of the parent entity).  Since the type of the expected argument is generated for you (`QueryGetBusinessInfoArgs`) you can call `args` something more specific to what you're expecting.

Reply:
The type shouldn't get generated if there are no arguments defined for that query... if it is generated then it should be used.

Reply:
> hm maybe there's a better type for that, but we don't use it anyway

bumping this because it's still somewhat relevant in the updated code, the `_` (`parent`) argument has `any` as its type